### PR TITLE
used stripBOM module instead of a function

### PR DESCRIPTION
**Pull request by noamokman**
Hi!
Just replaced a `stripBOM` function with an [existing module](https://github.com/sindresorhus/strip-bom).
I tried to run the tests, but they failed before I changed anything.

Anything I am missing?

Thanks!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -171,6 +172,7 @@
     "slide",
     "sorted-object",
     "strip-ansi",
+	"strip-bom",
```
Comment (sindresorhus) [inline package.json]: Wrong indent

**Comment by sindresorhus** [comment]
npm bundles all dependencies, so you have to do so with this dependency too.

**Comment by othiym23** [comment]
Is there a particular reason to switch to the module? The current implementation is dirt simple and documents exactly what it's doing (and I doubt it's going to need to change without something extraordinary happening).

**Comment by noamokman** [comment]
@sindresorhus What do you mean? 
> npm bundles all dependencies, so you have to do so with this dependency too.

@othiym23 This [module](https://github.com/sindresorhus/strip-bom) does the same,
The current implementation has this comment:
```js
// from read-package-json
```
This helps reduce the npm code base.

File: package.json (hunk 1 above)
Comment (noamokman) [inline package.json]: Opps :sweat_smile: fixed in  6d69890

**Comment by othiym23** [comment]
> We should strive for more modularization.
> This helps reduce the npm code base.

I think this is a good general principle, but in this case, the amount of functionality replaced is too small to be worth the added dependency. We're getting to the point where there's so much startup time caused by npm reading all its source files that I'm starting to be more careful about splitting things up into smaller pieces or adding more dependencies. Because of that, I'm not taking this patch. Thanks for putting this together and for answering my questions!

**Pull request closed without merging**